apps/website/src/components/institute/EditPayPalPixels.tsx

```diff
@@ -0,0 +1,162 @@
+    DonationPixel[] | undefined
+  >(undefined);
+  const getPayPalPixels = trpc.donations.getPayPalPixels.useMutation({
+    onSuccess: (data) => {
+      setLastPixelsData(data);
+    },
+  });
+
+  const [overrideIdentifier, setOverrideIdentifier] = useState("");
+  const [defaultIdentifier, setDefaultIdentifier] = useState("Anonymous");
+
+  const verificationRef = useRef<PayPalVerification | null>(null);
+
+  const submit = async (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    const formData = new FormData(e.currentTarget);
+    const firstName = String(formData.get("firstName"));
+    const lastName = String(formData.get("lastName"));
+    const email = String(formData.get("email"));
+
+    const input = payPalVerificationSchema.safeParse({
+      firstName,
+      lastName,
+      email,
+    });
+
+    setDefaultIdentifier(firstName);
+    // If we search new, we do not want to keep old pixel data
+    setLastPixelsData(undefined);
+
+    if (input.success) {
+      verificationRef.current = input.data;
+      getPayPalPixels.mutate({
+        muralId,
+        verification: input.data,
+      });
+    } else {
+      verificationRef.current = null;
+    }
+  };
+
+  const persistedPixelsData = getPayPalPixels.data ?? lastPixelsData;
+  const pixelsQuery = useMemo(
+    () => ({
+      ...getPayPalPixels,
+      data: persistedPixelsData,
+    }),
+    [getPayPalPixels, persistedPixelsData],
+  );
+
+  const renameMutation = trpc.donations.renamePayPalPixels.useMutation();
+
+  return (
+    <>
+      <form onSubmit={submit}>
+        <div className="mb-4 space-y-4">
+          <div className="max-w-xl space-y-4">
+            <Fieldset legend="Your PayPal donation info">
+              <TextField
+                label="First name(s)"
+                name="firstName"
+                autoComplete="given-name"
+                isRequired={true}
+              />
+
+              <TextField
+                label="Last name(s)"
+                name="lastName"
+                autoComplete="family-name"
+                isRequired={true}
+              />
+
+              <TextField
+                label="Email address"
+                name="email"
+                type="email"
+                autoComplete="email"
+                isRequired={true}
+              />
+            </Fieldset>
+
+            <FormButton size="small" type="submit">
+              Search
+            </FormButton>
+          </div>
+
+          {getPayPalPixels.isPending ? (
+            <MessageBox>Searching for your pixels…</MessageBox>
+          ) : null}
+
+          {getPayPalPixels.isError ? (
+            <MessageBox variant="failure">
+              Failed to find your pixels: {getPayPalPixels.error.message}
+            </MessageBox>
+          ) : null}
+
+          {getPayPalPixels.isSuccess && getPayPalPixels.data.length < 1 ? (
+            <MessageBox variant="warning">
```

Comment:
Should this have a border to be consistent with the cards that show when you do have pixels?